composer.json

```diff
@@ -19,7 +19,8 @@
 	],
 
 	"require" : {
-		"php" : ">=5.4.8",
+		"php" : ">=5.6",
+		"phpunit/phpunit" : "5.*",
```

Comment:
This should probably be put this under "require-dev" as to avoid including it in production.